packages/jobs/lib/crons/autoIdleDemo.ts

```diff
@@ -0,0 +1,87 @@
+import { schedule } from 'node-cron';
+import {
+    CommandToActivityLog,
+    ErrorSourceEnum,
+    SyncClient,
+    SyncCommand,
+    createActivityLog,
+    createActivityLogMessageAndEnd,
+    errorManager,
+    updateSuccess as updateSuccessActivityLog,
+    updateScheduleStatus,
+    isErr,
+    findPausableDemoSyncs
+} from '@nangohq/shared';
+import tracer from 'dd-trace';
+import { SpanTypes } from '@nangohq/shared/lib/utils/metrics.manager';
+
+export async function cronAutoIdleDemo(): Promise<void> {
+    schedule('1 * * * *', () => {
+        const span = tracer.startSpan(SpanTypes.JOBS_IDLE_DEMO);
+        tracer.scope().activate(span, async () => {
+            try {
+                await exec();
+            } catch (err: unknown) {
+                const e = new Error('failed_to_auto_idle_demo', { cause: err instanceof Error ? err.message : err });
+                errorManager.report(e, { source: ErrorSourceEnum.PLATFORM }, tracer);
+            }
+            span.finish();
+        });
+    });
+}
+
+export async function exec(): Promise<void> {
+    console.log('[autoidle] starting');
+
+    const syncs = await findPausableDemoSyncs();
+
+    console.log(`[autoidle] found ${syncs.length} syncs`);
```

Comment:
Not a blocker, but we should move to use the logger instead of raw console logs https://github.com/NangoHQ/nango/blob/master/packages/server/lib/utils/logger.ts#L8

Reply:
ah yes I didn't know since `console.log` is everywhere, will change 👍🏻